backends/sycl/ceed-sycl-compile.sycl.cpp

```diff
@@ -77,24 +77,34 @@ static inline int CeedJitCompileSource_Sycl(Ceed ceed, const sycl::device &sycl_
 
 // ------------------------------------------------------------------------------
 // Load (compile) SPIR-V source and wrap in sycl kernel_bundle
-// TODO: determine appropriate flags
-// TODO: Error handle lz calls
 // ------------------------------------------------------------------------------
-static int CeedJitLoadModule_Sycl(const sycl::context &sycl_context, const sycl::device &sycl_device, const ByteVector_t &il_binary,
+static int CeedJitLoadModule_Sycl(Ceed ceed, const sycl::context &sycl_context, const sycl::device &sycl_device, const ByteVector_t &il_binary,
```

Comment:
I feel like the name of this function should be matched with the others (ie. `CeedLoadModule_Sycl`), but I'm not sure.

Reply:
Will modify the function name for this. But I hope using the 'Jit' tag in the function name for the others above are OK, since they are specific to the SYCL backend and provide a better sense of what is done in that function.

Reply:
Then perhaps it's not necessary to change this function name. I'm not as familiar with the other backends to know whether these functions have equivalents in CUDA or HIP. @jeremylt and/or @nbeams would know better.

Reply:
CUDA and HIP also do JiT. We should use the equivalent function name from those backends but with a different suffix

Reply:
Yeah, looks like the sporadic use of JiT in function names was removed from the other GPU backends